changelog.d/5856.misc

```diff
@@ -0,0 +1 @@
+Tag authenticated entity and servlet_name.
```

Comment:
more context needed here please. Remember this is to be read by random synapse admins in the community...

Reply:
also, could be a `.feature` ?

Reply:
I didn't know who read the newsfiles. 

The change is so small does it warrant feature? What would constitute as misc?

Reply:
the newsfiles get turned directly into entries in the CHANGES.md file, which then go into the release notes (https://github.com/matrix-org/synapse/releases/tag/v1.3.0 etc).

"misc" is really intended for refactors that have no functional effect.

I'd say nothing is "too small" to be a feature, though it might make sense to group together a bunch of things (eg "Various opentracing-related improvements"), which can be done by giving them the same text in the changelog.d file. 

I think what you've done now is fine though.

Reply:
Oh, that's cool. I'll keep that in mind